iOSNativeLoginTemplate/iOSNativeLoginTemplate/SceneDelegate.swift

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright (c) 2019-present, salesforce.com, inc. All rights reserved.
+
+Redistribution and use of this software in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+* Redistributions of source code must retain the above copyright notice, this list of conditions
+and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright notice, this list of
+conditions and the following disclaimer in the documentation and/or other materials provided
+with the distribution.
+* Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
+endorse or promote products derived from this software without specific prior written
+permission of salesforce.com, inc.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import UIKit
+import SwiftUI
+import MobileSync
+import SalesforceSDKCore
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+    
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        self.window = UIWindow(frame: windowScene.coordinateSpace.bounds)
+        self.window?.windowScene = windowScene
+        
+        // This Client Id and Redirect Url is specific and exclusive to Native Login.
+        let clientId = "3MVG9CEn_O3jvv0wTqRT0Le6tmzX.EQ9ZvtHL1TG3gHFV.4IvKZyXw5SgdiVPi61mXrpu40mCOhKevEfYNMOm"
```

Comment:
Will customer be able to login to their org or will they need to first edit those lines (after doing the necessary setup in their org)?

Reply:
@wmathurin They will need to define their own so I will definitely update the comment here to state that.  Should I remove our values to make it more obvious?  (I could even Assert the are not empty).  If I leave them it makes testing easier for us but not that big of a deal.  CI can work around this too.

Reply:
Could be something like this:
```
        ///
        /// Fill in the values below from the connected app that was created for Native Login and
        /// the url of your Experience Cloud community.
        ///
        let clientId = "your-client-id"
        let redirectUri = "your-redirect-uri"
        let loginUrl = "your-community-url"
        
        assert(clientId != "your-client-id", "Please add your Native Login client id.")
        assert(redirectUri != "your-redirect-uri", "Please add your Native Login redirect uri.")
        assert(loginUrl != "your-community-url", "Please add your Native Login community url.")
```

We can also update it with a link to the documentation when it goes live.

Reply:
We might need to amend the next steps instructions that forceios prints out.
Right now it only mentions the bootconfig file:
https://github.com/forcedotcom/SalesforceMobileSDK-Package/blob/dev/shared/createHelper.js#L237